runtime-attach/src/main/java/io/opentelemetry/contrib/attach/RuntimeAttach.java

```diff
@@ -32,30 +30,32 @@ public static void attachJavaagentToCurrentJVM() {
     ByteBuddyAgent.attach(javaagentFile, getPid());
 
     if (!agentIsAttached()) {
-      logger.warning("Agent was not attached. An unexpected issue has happened.");
+      printError("Agent was not attached. An unexpected issue has happened.");
     }
   }
 
+  @SuppressWarnings("SystemOut")
+  private static void printError(String message) {
+    System.err.println(message);
```

Comment:
```suggestion
    // not using java.util.logging in order to avoid initializing the global LogManager
    // too early (and incompatibly with the user's app),
    // and because this is too early to use the Javaagent's PatchLogger
    System.err.println(message);
```

Reply:
I'm not sure whether mentioning PatchLogger here makes sense as this code is independent from the java agent.

Reply:
can you clarify what you mean by independent from the java agent? from my perspective I think of it as very dependent since it is attaching the java agent at runtime